docs/workflow/building/libraries/README.md

```diff
@@ -75,10 +75,10 @@ The above commands will give you libraries in "debug" configuration (the default
 
 The libraries build has two logical components, the native build which produces the "shims" (which provide a stable interface between the OS and managed code) and the managed build which produces the MSIL code and NuGet packages that make up Libraries. The commands above will build both.
 
-The build settings(TargetFramework, OSGroup, Configuration, Architecture) are generally defaulted based on where you are building (i.e. which OS or which architecture) but we have a few shortcuts for the individual properties that can be passed to the build scripts:
+The build settings(TargetFramework, BuildOS, Configuration, Architecture) are generally defaulted based on where you are building (i.e. which OS or which architecture) but we have a few shortcuts for the individual properties that can be passed to the build scripts:
 
 - `-framework|-f` identifies the target framework for the build. Possible values include `netcoreapp5.0` (currently the latest .NET Core version) or `net472`. (msbuild property `BuildTargetFramework`)
-- `-os` identifies the OS for the build. It defaults to the OS you are running on but possible values include `Windows_NT`, `Unix`, `Linux`, or `OSX`. (msbuild property `OSGroup`)
+- `-os` identifies the OS for the build. It defaults to the OS you are running on but possible values include `Windows_NT`, `Unix`, `Linux`, or `OSX`. (msbuild property `BuildOS`)
```

Comment:
I think `-os` should set HostOS instead i.e. the OS where the built libraries will eventually execute.

We seem to have some inconsistency in the build where sometimes BuildOS is incorrectly used for that instead.

Reply:
> OS where the built libraries will eventually execute.

This should be TargetOS

Reply:
It depends on which terminology you're using. In GNU's Canadian-Cross interpretation we have `build` (== where the build of binary is running), `host` (== where the binary itself will be running) and `target` (== where the code generated by the binary will be running).

I guess in this context it makes sense to just use build + target.